src/services/school/hooks/index.js

```diff
@@ -1,6 +1,8 @@
 const { authenticate } = require('@feathersjs/authentication');
+const { Forbidden } = require('@feathersjs/errors');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to resolve path to module '@feathersjs/errors'.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4605061)